main.go

```diff
@@ -20,11 +20,21 @@ package main
 import (
 	"github.com/dispatchlabs/disgo/bootstrap"
 	"github.com/dispatchlabs/disgo/commons/utils"
+	"github.com/dispatchlabs/disgo/commons/types"
 )
 
+var version string
+var date string
+
 func main() {
 	utils.InitMainPackagePath()
 	utils.InitializeLogger()
+	if version == "" {
+		utils.Error("You have not specified a version when built, please see documentation.")
```

Comment:
We have a "getDefaultVersion" method. Should we just call that in the null case instead of throwing an error?

Reply:
Yes, we can absolutely do that.  In this case, it was just logging, not throwing an error.  Downstream if it is empty it should be using the default version if it wasn't specified.  I will remove the log error here.